### DISCUSSION TITLE
Use something like shortenCommandLine

### DISCUSSION THREAD

**Discussion by erikson84** [rcasia/neotest-java#170] [Q&A] [answered]
Hi!

I'm working on a large Java project on Neovim Windows, and I encountered  many problems with jdtls and java-debug until I found out about the 

> shortenCommandLine: 'argfile'

argument, set either in the .vscode/launch.json or in the jdtls configuration (see [this discussion](https://github.com/mfussenegger/nvim-jdtls/issues/674#issuecomment-2229469899)).

I am experimenting with neotest-java for better test handling within Neovim, but I'm getting the usual issues that happens when I try to run or debug without the shortenCommandLine argument set to 'argfile': lockups, timeouts and no output.

My configuration for neotest-java is the same as the LazyVim setup described in the README.

OS: Windows 11 Pro (23H2)
Neovim: 0.11
JDTLS: 1.43
java-test: 0.43
LazyVim: 14.6.0

**Comment by rcasia** [comment, marked as answer]
Moved to https://github.com/rcasia/neotest-java/issues/182
